js/sigin.js

```diff
@@ -0,0 +1,82 @@
+const emailInput = document.getElementById("emailInput");
+const noEmail = document.getElementById("noEmail");
+const wrongEmail = document.getElementById("wrongEmail");
+
+emailInput.addEventListener("focusout", function () {
+  const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+
+  if (!emailInput.value.trim()) {
+    // 이메일을 입력해주세요 에러 메세지
+    noEmail.style.display = "block";
+    wrongEmail.style.display = "none";
+    emailInput.style.borderColor = "var(--red)";
+  } else if (!emailRegex.test(emailInput.value)) {
+    // 이메일 형식 오류 에러 메세지
+    noEmail.style.display = "none";
+    wrongEmail.style.display = "block";
+    emailInput.style.borderColor = "var(--red)";
+  } else {
+    noEmail.style.display = "none";
+    wrongEmail.style.display = "none";
+    emailInput.style.borderColor = "var(--gray20)";
+  }
+});
+
+const passwordInput = document.getElementById("passwordInput");
+const noPassword = document.getElementById("noPassword");
+
+passwordInput.addEventListener("focusout", function () {
+  if (!passwordInput.value.trim()) {
+    // 비밀번호를 입력해주세요 에러 메세지
+    noPassword.style.display = "block";
+    passwordInput.style.borderColor = "var(--red)";
+  } else {
+    noPassword.style.display = "none";
+    passwordInput.style.borderColor = "var(--gray20)";
+  }
+});
+
+const checkEmail = document.getElementById("checkEmail");
+const checkPassword = document.getElementById("checkPassword");
+
+function checkAccount() {
```

Comment:
[여기](https://github.com/codeit-bootcamp-frontend/2-Weekly-Mission/pull/160#discussion_r1391118138)에서 event 객체를 전달했으니 checkAccount 함수의 선언부도 아래와 같이 변경하고, 폼의 기본 동작을 방지하기 위해 preventDefault 함수도 호출해주어야겠네요
```suggestion
function checkAccount(event) {
  event.preventDefault();
```